### PR TITLE
[JDK-8230775] add release notes for JavaFX 13 release

### LINKED ISSUE #586: JDK-8230775: release notes for JavaFX 13
See https://bugs.openjdk.java.net/browse/JDK-8230775

### PR DESCRIPTION
**Pull request by johanvos**
This fixes #586

### CODE REVIEW

File: doc-files/release-notes-13.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## List of Enhancements
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8226454](https://bugs.openjdk.java.net/browse/JDK-8226454)|Point2D and Point3D should implement Interpolatable|animation
+[JDK-8221269](https://bugs.openjdk.java.net/browse/JDK-8221269)|Extract embedded actions from JSL grammar file to Visitor class|build
+[JDK-8223760](https://bugs.openjdk.java.net/browse/JDK-8223760)|support static build for macosx|build
+[JDK-8222258](https://bugs.openjdk.java.net/browse/JDK-8222258)|Add exclusion scope for LightBase|graphics
+[JDK-8167148](https://bugs.openjdk.java.net/browse/JDK-8167148)|Add native rendering support  by supporting WritableImages backed by NIO ByteBuffers|graphics
```
Comment (nlisker) [inline doc-files/release-notes-13.md]: Somehow there's an extra space in "support  by".

Comment (kevinrushforth) [reply doc-files/release-notes-13.md]: Oops. This one is my fault...I see that it was in the data I pulled from JBS and sent to Joeri and Johan.

**Comment by kevinrushforth** [comment]
I think this PR can be closed now.

**Comment by kevinrushforth** [comment]
(we don't want to merge it into jfx-13, since it was already pushed via HG as per normal)

**Pull request closed without merging**